server/src/routes/api/v1/shoesRouter.js

```diff
@@ -8,6 +8,18 @@ const shoesRouter = new express.Router()
 
 shoesRouter.use("/:shoeId/reviews", shoeReviewsRouter)
 
+shoesRouter.delete("/:id", async (req, res) => {
+  const { body } = req
+  try {
+    const shoe = await Shoe.query().findById(body.id)
+    const deletedReviews = await shoe.$relatedQuery("reviews").delete()
```

Comment:
deleting the associated reviews should be an action handled by the database - as this application grows and we add more associations, having to manage deleting them all is too much overhead. Instead we should add an `onDelete("CASCADE")` to the reviews table - this can be added in a new migration so you don't have to rollback and edit an existing migration

you can see a question from another group here in the comments to see how to set up the new migration:
https://learn.launchacademy.com/questions/14055

once added, we can remove line 15 because the database will delete associated records for us